README.md

```diff
@@ -9,6 +9,7 @@ The following environment variables are available:
 | Name | Example Value | Required |
```

Comment:
Is there a place in the doc that explains how to use the applied?

Reply:
Added!